index.html

```diff
@@ -38,7 +38,7 @@ <h4 class="modal-title text-center" id="myModalModal">ML Platform</h4>
     <div class="container text-center">
         <h2 class="text-center">What is the machine learning platform?</h2>
         <p>
-            The machine learning platform is part of the <a href="https://www.linaro.org/news/linaro-announces-launch-of-machine-intelligence-initiative/">Linaro Machine Intelligence Initiative</a> and is the home for Arm NN and Arm
+            The machine learning platform is part of the <a href="https://www.linaro.org/news/linaro-announces-launch-of-machine-intelligence-initiative/">Linaro Machine Artificial Initiative</a> and is the home for Arm NN and Arm
```

Comment:
```suggestion
            The machine learning platform is part of the <a href="https://www.linaro.org/news/linaro-announces-launch-of-machine-intelligence-initiative/">Linaro Artificial Intelligence Initiative</a> and is the home for Arm NN and Arm
```
??

Reply:
Yep this was changed a while back just needed updating
https://www.linaro.org/engineering/artificial-intelligence/

Reply:
But you seem to have a typo in the name (?), hence my suggestion.